src/expression.d

```diff
@@ -15599,8 +15599,8 @@ extern (C++) final class IdentityExp : BinExp
         if (e1.type.toBasetype().ty == Tsarray ||
             e2.type.toBasetype().ty == Tsarray)
             deprecation("identity comparison of static arrays "
-                "implicitly coerces them to slices, "
-                "which are compared by reference");
+                ~ "implicitly coerces them to slices, "
+                ~ "which are compared by reference");
```

Comment:
My personal preference is to put operators at the end of the line in case of expressions spanning multiple lines, but I want insist on that, since it's a nitpick and I'm not sure on the prevailing usage in dmd's code anyway.